src/ankita/DuplicateFromArray.java

```diff
@@ -0,0 +1,34 @@
+package ankita;import java.util.Arrays;
+
+import suresh.arrayExample4;
+
+public class DuplicateFromArray {
+	void getDuplicateFromIntArray(int[] arr) {
+		System.out.print("Duplicate from Int array:");
+		for (int i = 0; i < arr.length; i++) {
+				for (int j = i+1; j < arr.length; j++) {
```

Comment:
{ 10, 12,32, 55, 32, 17, 12, 32,32 } try this input. code wont work as expected

Reply:
yes. Will Correct the code.